installer/ovatools/ovfenv/main.go

```diff
@@ -44,6 +44,10 @@ func main() {
 			Value: "",
 			Usage: "Get single OVF property",
 		},
+		cli.BoolFlag{
+			Name:  "dump",
+			Usage: "Dump the OVF Environment XML",
```

Comment:
yep, dumps the XML and quits, useful for debugging when the go parser fails to handle proper xml 😄